.gitignore

```diff
@@ -1,3 +1,4 @@
 /.project
 target/
-/site/public/
\ No newline at end of file
+/site/public/
+site/.hugo_build.lock
```

Comment:
I built the site with Hugo in GitHub Codespaces and it created this file for me. It's not something I want to commit, and I thought it's best to ignore it, to avoid any confusion for someone else.